src/Aspire.Hosting.Azure.ServiceBus/ApplicationModel/ServiceBusClientAffineProperties.cs

```diff
@@ -0,0 +1,51 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Aspire.Hosting.Azure.ServiceBus.ApplicationModel;
+
+/// <summary>
+/// Properties specific to client affine subscriptions.
+/// </summary>
+public class ServiceBusClientAffineProperties
+{
+    private readonly OptionalValue<string> _clientId = new();
+    private readonly OptionalValue<bool> _isDurable = new();
+    private readonly OptionalValue<bool> _isShared = new();
+
+    /// <summary>
+    /// Creates a new ServiceBusClientAffineProperties.
+    /// </summary>
+    public ServiceBusClientAffineProperties()
+    {
+    }
+
+    /// <summary>
+    /// Indicates the Client ID of the application that created the
+    /// client-affine subscription.
+    /// </summary>
+    public OptionalValue<string> ClientId
+    {
+        get { return _clientId!; }
+        set { _clientId!.Assign(value); }
```

Comment:
Why is the call to `Assign` needed? Looks like the implicit operator would handle this without the need for a backing field at all.

Reply:
> Looks like the implicit operator would handle this

Maybe. It was done before I update the operator. I'd keep it this way to make it obvious.

Reply:
OptionalValue was removed